lib/util.js

```diff
@@ -185,7 +185,7 @@ util.getQS = function(allowedProps,args) {
         , qs;
 
     for(var i = 0; i < allowedProps.length; i++) {
-        if(args[allowedProps[i]])
+        if(args.hasOwnProperty([allowedProps[i]]))
```

Comment:
I think that it should be `if(args.hasOwnProperty(allowedProps[i]))`. Please double-check.

Reply:
I can confirm that @vlavrynovych 's suggestion works for me in another scenario. I also would really like to see this pushed because it is very helpful for a useful trick in leaderboard searches.